src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -291,11 +296,12 @@ public async ValueTask<IDbCommand> CreateAsync(CancellationToken cancellationTok
 			return DoCreate(connection);
 		}
 
-		internal DbConnectorCommand(DbConnector connector, string text, DbParameters parameters, bool isCached = false)
+		internal DbConnectorCommand(DbConnector connector, string text, DbParameters parameters, bool isStoredProcedure = false, bool isCached = false)
```

Comment:
It would be simpler to use `CommandType commandType = CommandType.Text` here, especially when cloning the command, as in line 275.